djstripe/mixins.py

```diff
@@ -40,9 +40,8 @@ def get_context_data(self, **kwargs):
         context = super(PaymentsContextMixin, self).get_context_data(**kwargs)
         context.update({
```

Comment:
Go ahead and change this to match the get_context_data in other classes by using ``context['variable'] = value`` instead of ``context.update``.